pysteps/blending/steps.py

```diff
@@ -65,1723 +64,2862 @@
 except ImportError:
     DASK_IMPORTED = False
 
+from dataclasses import dataclass, field
+from typing import Optional, List, Dict, Any, Callable, Union
```

Comment:
`Optional`, `List`, `Dict` and `Union` are not necessary to import for type hints since python 3.10.
`List` and `Dict` can be replaced with `list` and `dict`
and something like `Optional[Union[list, np.ndarray]]` could be written as `list | np.ndarray | None`

Reply:
Thanks Mats! I am clearly not that well versed in the typing in python and must have used an older resource when I was writing this. Ill update it later this week :)

Reply:
Should be fixed in the new version